spec/index.html

```diff
@@ -576,7 +576,7 @@ <h3>IRIs</h3>
 
     <p>An <dfn data-lt="iri"><abbr title="Internationalized Resource Identifier">IRI</abbr></dfn>
       (Internationalized Resource Identifier) within an RDF graph
-      is a Unicode string [[!UNICODE]] that conforms to the syntax
+      is a unicode string [[!UNICODE]] that conforms to the syntax
```

Comment:
Are we going with `Unicode` (which appears to me to be correct) or `unicode`? And are we using one form throughout the documents, or changing for some reason(s) that are not clear to me?

This PR currently has a mix of `Unicode` and `unicode` in the human-facing text.

I think we should be using one form throughout the documents, and that it should have the capital `U`, i.e., `Unicode` (as I note is used throughout the [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/), not to mention [the Unicode website](https://home.unicode.org/)).

Reply:
Isn't it because, in this case, "unicode string" is informal (local) terminology?

Reply:
The proximal `[[!UNICODE]]` citation suggests it's not so informal.

In such case, I would hope for a glossary entry or similar for "unicode string" that would differentiate it from "Unicode" and/or "Unicode other things"...

I also note [The Unicode® Consortium Name and Trademark Usage Policy](https://www.unicode.org/policies/logo_policy.html) with which I think we should comply, unless there's a strong argument not to (and W3C legal OKs such non-compliance).

Reply:
It is unfortunate that there is no good reference for a "string" term; i18n-glossary says its intended to be used along INFRA, but there a string is defined based on JavaScript, which isn't helpful.

I agree with @TallTed that we should be consistent and use "Unicode string" uniformly; I can add a commit to do so here. In lieu of a term reference, we can make sure we're using [[UNICODE]] in addition, where appropriate.

Reply:
I think we can mark this thread resolved.